homeassistant/components/template/number.py

```diff
@@ -70,7 +70,7 @@
         vol.Required(CONF_NAME): cv.template,
         vol.Required(CONF_STATE): cv.template,
         vol.Required(CONF_STEP): cv.template,
-        vol.Optional(CONF_SET_VALUE): cv.SCRIPT_SCHEMA,
+        vol.Required(CONF_SET_VALUE): cv.SCRIPT_SCHEMA,
```

Comment:
I think we can clean this up now:
https://github.com/home-assistant/core/blob/5cc7e6667c944460910f493675300941cdd6f92f/homeassistant/components/template/number.py#L159